content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -192,51 +235,75 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 
 ### Install kubectl binary with curl on Windows
 
-1. Download the latest release {{< param "fullversion" >}} from [this link](https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
+1. Download the latest release, {{< param "fullversion" >}}, from [this link](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
 
-   Or if you have `curl` installed, use this command:
+    Or if you have `curl` installed, use this command:
```

Comment:
nit: Check that the line "Or if .." is lined up under the "D" in Download?

Reply:
I've made a mistake updating the spacing in this file. I'll go through and pick out these extra spaces.